src/restic/backend/rest/backend_test.go

```diff
@@ -0,0 +1,87 @@
+// DO NOT EDIT, AUTOMATICALLY GENERATED
```

Comment:
Should the source and the generation recipe for the auto-generation be part of the repo?